package.json

```
@@ -10,7 +10,7 @@
   "packageManager": "yarn@3.4.1",
   "private": true,
   "scripts": {
-    "build": "tsc --noEmit && ts-node ./esbuild.config.ts",
+    "build": "mkdir -p dist && cp -f open-api-specification.yaml dist/open-api-specification.yaml && tsc --noEmit && ts-node ./esbuild.config.ts",
```

V minor thing, but you can chain yaml scripts if you take bits out into their own scripts.

So we could have `yarn bundleApiSpecs`, which did `mkdir -p dist && cp -f open-api-specification.yaml dist/open-api-specification.yaml`

Then this line could be:
```suggestion
    "build": "yarn bundleApiSpecs && tsc --noEmit && ts-node ./esbuild.config.ts",
```

---

Just makes these things easier to read and keeps them from getting too long.

---

Cool didn't know that was a thing! I've added it in